src/components/item/table-head.js

```diff
@@ -92,6 +92,15 @@ class ItemTableHead extends PureComponent {
       <table className="table-head">
         <thead>
           <tr>
+            {this.props.showPositionColumn &&
+              <ItemTableHeadCell
+                key="position"
+                id="position"
```

Comment:
I don't know yet what it will look like exactly, but we'll have other 'special' columns soon. Let's define the id, label and width in a constant object `PositionColumn` for now.